packages/docs/static/definitions/processes/kycaml/0.0.1/generic--usa-kybp-accinv-all-checks

```diff
@@ -0,0 +1,29 @@
+{
+  "processName": "Accredited Investor",
+  "country": "USA",
+  "subjectType": "Legal entity",
+  "version": "0.0.1",
+  "basicChecks": [
+    "Legal Entity Name",
+    "Doing Business As Name",
+    "Legal Entity Address",
+    "Legal Entity State",
+    "Legal Entity Nation",
+    "Date of Formation",
+    "Tax ID/Registration Number",
+    "Existence Verified",
+    "Nature of business"
+  ],
+  "elegibilityChecks": [
+    "Assets",
+    "Income",
+    "Licensing"
```

Comment:
If this process is for a business then I think the two that would apply would be:
 1. Worth (at least $5M in assets)
 2. All of the owners of the business have gone through an accreditation check and are all currently accredited

Reply:
BOTH or EITHER have to be met?  

I'm also curious whether the per-owner checks listed below or the KYB checks listed above are overkill-- it might make sense for this credential to attest to LESS checks in those two categories but to require a separate KYB cred for anything where those checks are desired/required by relying party...

Reply:
At least one of the two criteria must be met for a business entity to be accredited.

RE the `AuthorizedRepresentativeChecks` - I'm not sure what that would indicate.  AFAIK there is no concept of an "authorized representative" who must also be "checked" for business accreditation.  Same thing for the `basicChecks` - it's not a requirement that the accreditation issuer verify the address or date of formation, for instance.  My suggestion would be to just remove those two fields to keep it centered on the basic accreditation test satisfied.